### PR TITLE
fix some missing semi-colons, and other general scss issues

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Found some rules that were using the same selector, that should be merged, found some properties that were missing semi-colons, and found one property that needed to be on two lines.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/css/components/_fullscreen.scss [hunk 1]
```diff
@@ -1,8 +1,6 @@
 .video-js .vjs-fullscreen-control {
   cursor: pointer;
   @include flex(none);
-}
-.video-js .vjs-fullscreen-control {
```
Comment (gkatsev) [inline src/css/components/_fullscreen.scss]: does `sass` output change for this?
But yeah, makes sense to have this.

Comment (BrandonOCasey) [inline src/css/components/_fullscreen.scss]: let me do a quick diff, but the byte size was exactly the same

Comment (BrandonOCasey) [inline src/css/components/_fullscreen.scss]: after doing the diff there are no changes

Comment (gkatsev) [inline src/css/components/_fullscreen.scss]: Sounds good. I didn't imagine it would.

**Comment by gkatsev** [comment]
Do we want to merge this in as a patch? If so, could you rebase this against stable? (you can change PR base branches in github now as well).

**Comment by gkatsev** [comment]
LGTM

**Comment by BrandonOCasey** [comment]
@gkatsev I don't think it matters where we do it, we can do it as a patch on stable if you want and I will make the changes

**Pull request closed without merging**